external_services/data.py

```diff
@@ -30,10 +31,12 @@ def get_data(endpoint: str, params: dict = None):
     if endpoint.startswith("http"):
```

Comment:
Can we try and move away from this as a way of checking if we're in test mode and move towards more mocking HTTP requests? Ideally the app code shouldn't have any idea about this switch.

Do realise this pre-dates this PR, so happy for it to be done in a subsequent PR.